docs/source/a_quick_tour.mdx

```diff
@@ -178,6 +179,23 @@ A common way to overcome this issue is to fallback on single process evaluation.
 
 This solution allows 🤗 Evaluate to perform distributed predictions, which is important for evaluation speed in distributed settings. At the same time, you can also use complex non-additive metrics without wasting valuable GPU or CPU memory.
 
+## Configuration
+
+Some metrics can be configured with additional settings. E.g. `accuracy` has the extra setting `normalize` which defaults to `True`. To change it you have two options: pass it as a keyword argument with `load` or during `compute`. With `load` the setting is changed permanently for the module while passing it to `compute` only changes it for the duration of the `compute` call.
```

Comment:
nit: in our other docs we usually aim to have parentheses for methods / functions to distinguish them from ordinary variables

```suggestion
Some metrics can be configured with additional settings. For example, `accuracy` has an extra `normalize` setting which returns the fraction of correctly classified samples and is set to `True` by default. To change it you have two options: pass it as a keyword argument with `load()` or during `compute()`. With `load()`, the setting is changed permanently for the module, while passing it to `compute()` only changes it for the duration of the `compute()` call.
```